integration-test/Css.ts

```diff
@@ -119,25 +119,25 @@ class CssBuilder<T extends Properties1> {
   display(value: Properties["display"]) { return this.add("display", value); }
 
   // flexbox
-  get justifyStart() { return this.add("justifyContent", "flex-start"); }
-  get justifyEnd() { return this.add("justifyContent", "flex-end"); }
-  get justifyCenter() { return this.add("justifyContent", "center"); }
-  get justifyBetween() { return this.add("justifyContent", "space-between"); }
-  get justifyAround() { return this.add("justifyContent", "space-around"); }
-  get justifyEvenly() { return this.add("justifyContent", "space-evenly"); }
-  justify(value: Properties["justifyContent"]) { return this.add("justifyContent", value); }
-  get selfStart() { return this.add("alignSelf", "flex-start"); }
-  get selfEnd() { return this.add("alignSelf", "flex-end"); }
-  get selfCenter() { return this.add("alignSelf", "center"); }
-  get selfBaseline() { return this.add("alignSelf", "baseline"); }
-  get selfStretch() { return this.add("alignSelf", "stretch"); }
-  self(value: Properties["alignSelf"]) { return this.add("alignSelf", value); }
-  get itemsStart() { return this.add("alignItems", "flex-start"); }
-  get itemsEnd() { return this.add("alignItems", "flex-end"); }
-  get itemsCenter() { return this.add("alignItems", "center"); }
-  get itemsBaseline() { return this.add("alignItems", "baseline"); }
-  get itemsStretch() { return this.add("alignItems", "stretch"); }
-  items(value: Properties["alignItems"]) { return this.add("alignItems", value); }
+  get js() { return this.add("justifyContent", "flex-start"); }
+  get je() { return this.add("justifyContent", "flex-end"); }
+  get jc() { return this.add("justifyContent", "center"); }
+  get jb() { return this.add("justifyContent", "space-between"); }
+  get ja() { return this.add("justifyContent", "space-around"); }
+  get jEvenly() { return this.add("justifyContent", "space-evenly"); }
+  j(value: Properties["justifyContent"]) { return this.add("justifyContent", value); }
```

Comment:
Maybe instead of just `j` this should be `jc` aka `justifyContent`? That would overlap with `jc`, but maybe all of the `js` / etc. should also really be `jcs`, `jce`, i.e. `Justify Content Start` --> `jcs`?

Reply:
I think it makes sense to be `jc_`. And `justify-content: center` would be `jcc`

Reply:
Cool, pushed that up